docs/kubernetes/overlays.md

```diff
@@ -0,0 +1,125 @@
+# Overview of per kubernetes minor version overlays
+
+The Persistent Disk CSI Driver manifest [bundle](../../deploy/kubernetes/base) is a set of many kubernetes resources (driver pod which includes the containers csi-provisioner, csi-resizer, csi-snapshotter, gce-pd-driver, csi-driver-registrar; csi driver object, rbacs, pod security policies etc). Not all driver capabilties can be used with all kubernetes versions. For example [volume snapshots](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) are supported on 1.17+ kubernetes versions. Thus structuring the overlays on a per kubernetes minor version is beneficial to quickly identify driver capabilities supported for a given minor version of kubernetes (example [stable-1-16](../../deploy/kubernetes/overlays/stable-1-16) driver manifests does not contain the snapshotter sidecar), and facilitates easy maintenance/updates of the CSI driver. Most up-to-date overlays is maintained in the master branch [overlays](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/tree/master/deploy/kubernetes/overlays) directory. Every stable-{k8s-version} overlay has its staging counterpart prow-gke-staging-release-rc-{k8s-version}.
```

Comment:
"Most up-to-date overlays is maintained in the master branch..."

Maybe more clear to invert that and say something like "The master branch has the up-to-date overlays for each kubernetes version in the [deploy/overlays](..) directory, which specify which driver release version to use with each kubernetes version."

Reply:
done